src/lib/components/layouts/Navigation.svelte

```diff
@@ -37,6 +37,8 @@
   $: BREAKPOINTS = {
     BOTTOM_NAV: width < remToPx(36),
   };
+
+  $: sign_in_url = new URL(`?next=${APP_URL}`, SIGN_IN_URL);
```

Comment:
Does this need to be reactive? Will this ever change?

Reply:
Realistically, it shouldn't. Maybe this moves to config at some point.